src/lib/ngsi/ContextElementResponse.cpp

```diff
@@ -268,7 +268,6 @@ ContextElementResponse::ContextElementResponse
     /* Setting custom metadata (if any) */
     if (attr.hasField(ENT_ATTRS_MD))
```

Comment:
Documentation missing (as discussed at skype... comenting in the PR to ensure we don't forget about it during weekend :)